recipe/meta.yaml

```
@@ -1,54 +1,168 @@
+        - icub-models
+        - gym-ignition-models
+        - idyntree
+        - git
+        - gitpython
+
+  - name: {{ name|lower }}-casadi
+    build:
+      noarch: python
+    requirements:
+      host:
+        - python  {{ python_min }}
+      run:
+        - {{ pin_subpackage('adam-robotics', max_pin='x.x.x') }}
+        - casadi
+        - python  >={{ python_min }}
+    test:
+      source_files:
+        - tests
+      commands:
+        - pytest ./tests/ -v -k "not gravity" --ignore-glob=*Jax* --ignore-glob=*pytorch*
+      requires:
+        - python  >={{ python_min }}
+        - pip
+        - pytest
+        - icub-models
+        - gym-ignition-models
+        - idyntree
+        - git
+        - gitpython
+        - casadi
+
+  - name: {{ name|lower }}-jax
+    build:
+      noarch: python
+    requirements:
+      host:
+        - python  {{ python_min }}
+      run:
+        - {{ pin_subpackage('adam-robotics', max_pin='x.x.x') }}
+        - jax
+        - python  >={{ python_min }}
+    test:
+      source_files:
+        - tests
+      commands:
+        - pytest ./tests/ -v -k "not gravity" --ignore-glob=*pytorch* --ignore-glob=*CasADi* --ignore-glob=*idyntree*
+      requires:
+        - python  >={{ python_min }}
+        - pip
+        - pytest
+        - icub-models
+        - gym-ignition-models
+        - idyntree
+        - git
+        - gitpython
+        - jax
+
+  - name: {{ name|lower }}-pytorch
+    build:
+      noarch: python
+    requirements:
+      host:
+        - python  {{ python_min }}
+      run:
+        - {{ pin_subpackage('adam-robotics', max_pin='x.x.x') }}
+        - jax2torch
+        - pytorch
+        - python  >={{ python_min }}
+    test:
+      source_files:
+        - tests
+      commands:
+        - pytest ./tests/ -v -k "not gravity" --ignore-glob=*Jax* --ignore-glob=*CasADi* --ignore-glob=*idyntree*
+      requires:
+        - python  >={{ python_min }}
+        - pip
+        - pytest
+        - icub-models
+        - gym-ignition-models
+        - idyntree
+        - git
+        - gitpython
+        - pytorch
+
+  - name: {{ name|lower }}-all
+    build:
+      noarch: python
+    requirements:
+      host:
+        - python  {{ python_min }}
+      run:
+        - {{ pin_subpackage('adam-robotics-casadi', max_pin='x.x.x') }}
+        - {{ pin_subpackage('adam-robotics-jax', max_pin='x.x.x') }}  # [not win]
+        - {{ pin_subpackage('adam-robotics-pytorch', max_pin='x.x.x') }}
+        - python  >={{ python_min }}
+    test:
+      source_files:
+        - tests
+      commands:
+        - pytest ./tests/ -v -k "not gravity"  # [not win]
+        - pytest ./tests/ -v -k "not gravity" --ignore-glob=*Jax*  # [win]
+      requires:
+        - python  >={{ python_min }}
+        - pip
+        - pytest
+        - icub-models
+        - gym-ignition-models
+        - idyntree
+        - git
+        - gitpython
+        - casadi
+        - pytorch
+        - jax  # [not win]
```

To avoid confusing the bot, perhaps we can remove the `# [win]` selectors, as anyhow the `-all` package at this point is not installable on Windows.
```suggestion
        - pytest ./tests/ -v -k "not gravity"
      requires:
        - python  >={{ python_min }}
        - pip
        - pytest
        - icub-models
        - gym-ignition-models
        - idyntree
        - git
        - gitpython
        - casadi
        - pytorch
        - jax
```

---

Thanks! Addressed in https://github.com/conda-forge/adam-robotics-feedstock/pull/12/commits/59c845cd57a37d4bc6d8bfb3371bb1529c05c3a5 with rebase